diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter6/classes/DataClassesRule.kt

```diff
@@ -10,11 +10,9 @@ import com.pinterest.ktlint.core.ast.ElementType.BLOCK
 import com.pinterest.ktlint.core.ast.ElementType.CLASS
 import com.pinterest.ktlint.core.ast.ElementType.CLASS_BODY
 import com.pinterest.ktlint.core.ast.ElementType.CLASS_INITIALIZER
-import com.pinterest.ktlint.core.ast.ElementType.DATA_KEYWORD
 import com.pinterest.ktlint.core.ast.ElementType.ENUM_KEYWORD
 import com.pinterest.ktlint.core.ast.ElementType.FUN
 import com.pinterest.ktlint.core.ast.ElementType.INNER_KEYWORD
-import com.pinterest.ktlint.core.ast.ElementType.INTERFACE_KEYWORD
```

Comment:
It's not the first time that I see how diktat removes unused imports in a file, on which it hasn't complained before. Do you have any thought, why this can happen?

Reply:
Yes. We need to investigate this

Reply:
Please create an issue about this too